clang-tools-extra/test/clang-tidy/performance-unnecessary-value-param.cpp

```diff
@@ -1,6 +1,31 @@
 // RUN: %check_clang_tidy %s performance-unnecessary-value-param %t
 
 // CHECK-FIXES: #include <utility>
+namespace std {
```

Comment:
Might be worth a comment here that this is the implementation of std::move

Reply:
Good point!. Adding comment.